spec/lib/manageiq/session_spec.rb

```diff
@@ -5,7 +5,7 @@
     def create_session
       sid, data = ManageIQ::Session.store.send(:find_session, request, nil)
       data["my_data"] = "stuff and things"
-      ManageIQ::Session.store.send(:write_session, request, sid, data)
+      ManageIQ::Session.store.send(:write_session, request, sid, data, {})
```

Comment:
Strange that we have .send here because these should be public methods (at least according to the Rails docs)

Reply:
TBH/ not sure why this change is in here